index.html

```diff
@@ -1395,33 +1397,33 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 extensions. This approach enables developers to easily experiment with new
 cryptographic signature mechanisms that support selective disclosure,
 zero-knowledge proofs, and post-quantum algorithms.
-          </dd>
-          <dt id="pf17b">
+            </dd>
+            <dt id="pf17b">
 PF17b: Supports signature chaining.
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 A signature chain is an ordered set of signatures over a data payload. Use
 cases, such as cryptographic signatures applied to a notarized document,
 typically require a signature by the signing party and then an additional one
 by a notary to be made after the original signing party has made their
 signature. Linked Data Proofs, including Linked Data Signatures, natively
 support chains of signatures. JWTs only enable a single signature over a
 single payload.
-          </dd>
-          <dt id="pf18b">
+            </dd>
+            <dt id="pf18b">
 PF18b: Does not require pre-processing or post-processing.
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 In order to encode a <a>verifiable credential</a> or a
 <a>verifiable presentation</a> in a JWT, an extra set of steps
 are required to convert the data to and from the JWT format. No such extra
 converstion step are required for <a>verifiable credentials</a> and
 <a>verifiable presentations</a> protected by LD-Proofs.
-          </dd>
-          <dt id="pf19b">
+            </dd>
+            <dt id="pf19b">
 PF19b: Canonicalization requires more than base-64 encoding.
```

Comment:
PF19b is currently not worded as a Benefit, but as a Cost -- and breaks from all the other items in the list.  It should be reworded --

`PF19b: Canonicalization requires more than base-64 encoding.`
-> `PF19b: Canonicalization does not require more than base-64 encoding.`

-- and the entry, checks, and x'es in the summary table (lines 1180-1184, in the current PR) should likewise be reversed, becoming --

```
<a href="#pf19b">PF19b.</a> Canonicalization does not require more than base-64 encoding.
                </td>
                <td class="missing">✖</td>
                <td class="missing">✖</td>
                <td class="supported">✓</td>
```

Reply:
fixed in [90a1587](https://github.com/w3c/vc-imp-guide/pull/28/commits/90a1587738291f5c434366809c153285996a1ae4)